complementary_standards.tex

```diff
@@ -4,6 +4,7 @@ \section{Complementary Standards}
 
 Here we discuss two complementary standards that have been adapted for use as part of SBOL representation, following the pattern for extension of SBOL described in Section~\ref{sec:Annotations}.
 In both cases, the extension uses the pattern in which object from another ontology are also assigned to either the SBOL \sbol{Identified} or \sbol{TopLevel} types.
+Note that this means that these that the object receives both an \external{rdf:type} for the SBOL class and also an \external{rdf:type} in their own namespace.
```

Comment:
`that these that the` -> `that the`

Sorry I didn't get to this before it got merged....